### PR TITLE
fixing header markup and some style tweaks

### PR DESCRIPTION
**Pull request by jasonkiss**
The two links are really buttons that perform actions. So don’t need
<nav>. Renamed relevant class names. Added clearer hover/focus styles
to buttons. Removed @aria-controls.

### CODE REVIEW

**Comment by jasonkiss** [comment]
Meant that I removed @aria-selected. Left @aria-controls as it is hook used to grab appropriate dialog container. Still need to tweak the modal dialog script: don't think we should have @aria-controls, and need to make dialog actually modal, so trap keyboard focus. I've got a working script that I'll try to implement over next while.

**Comment by Aupajo** [comment]
Just merged #5 – can you rebase on to master, @jasonkiss?

File: themes/bento/public/stylesheets/_header.scss [hunk 1]
```diff
@@ -42,8 +42,17 @@
   }
 }
 
-.headerLink {
```
Comment (jamesmacfie) [inline themes/bento/public/stylesheets/_header.scss:45]: Can I please get you to change this to:

``` scss
.headerAction {
  background: none;
  border: none;
  color: $headerText;
  font-weight: bold;
  padding: 2px 1px 1px;
  text-decoration: underline;

  &:focus, &:hover {
    background: white;
    color: $primary;
    text-decoration: none;
  }
}

```

It'll compile to the same thing. Chur

**Comment by jamesmacfie** [comment]
> The two links are really buttons that perform actions. So don’t need. Renamed relevant class names

Nice. I like your thinking

**Comment by jamesmacfie** [comment]
Sorry, merge conflict!

**Comment by jasonkiss** [comment]
This PR can be killed as it was dealt with as part of subsequent PR #8

**Pull request closed without merging**